src/store/index.js

```diff
@@ -9,6 +9,7 @@ import modals from './plugins/modals';
 import tipUrl from './plugins/tipUrl';
 import accounts from './plugins/account';
 import tokens from './plugins/tokens';
+import namesModule from './plugins/names';
```

Comment:
```suggestion
import names from './plugins/names';
```
actually, it is a plugin, not a module